src/AppInstallerCLICore/Workflows/UninstallFlow.cpp

```diff
@@ -124,13 +124,16 @@ namespace AppInstaller::CLI::Workflow
         context.Reporter.Info() << Resource::String::UninstallFlowUninstallSuccess << std::endl;
     }
     
-    void GetDependencies(Execution::Context& context)
+    void GetDependenciesInfo(Execution::Context& context)
     {
-        const auto& installer = context.Get<Execution::Data::Installer>();
-        if (installer && installer->Dependencies.HasAny())
+        const auto& installedPackageVersion = context.Get<Execution::Data::InstalledPackageVersion>();
+        const auto& installerOptions = installedPackageVersion->GetManifest().Installers;

```

Comment:
This doesn't work (yet); you need to follow the pattern I gave in an older comment for the current codebase to get a manifest to work with.